src/tools/crossgen2/Common/JitInterface/JitConfigProvider.cs

```diff
@@ -21,18 +21,20 @@ public IntPtr UnmanagedInstance
             get;
         }
 
+        public string JitPath
+        {
+            get;
+        }
+
         public IEnumerable<CorJitFlag> Flags => _jitFlags;
 
         /// <summary>
         /// Creates a new instance of <see cref="JitConfigProvider"/>.
         /// </summary>
         /// <param name="parameters">Name-value pairs separated by an equals sign.</param>
-        public JitConfigProvider(IEnumerable<CorJitFlag> jitFlags, IEnumerable<KeyValuePair<string, string>> parameters)
+        public JitConfigProvider(string jitPath, IEnumerable<CorJitFlag> jitFlags, IEnumerable<KeyValuePair<string, string>> parameters)
```

Comment:
Nit: this makes it feel like `jitPath` is a mandatory parameter. It could be a separate constructor (or at least a parameter with a null default value), since specifying JitPath is optional.

Reply:
I agree. I made `jitPath` the last parameter and optional.